src/control/controlmodel.cpp

```diff
@@ -16,11 +16,19 @@ ControlModel::ControlModel(QObject* pParent)
     const QList<QSharedPointer<ControlDoublePrivate>> controlsList =
             ControlDoublePrivate::getAllInstances();
 
+    const auto aliases = ControlDoublePrivate::getControlAliases();
+    QSet<ConfigKey> controlKeys(aliases.cbegin(), aliases.cend()); 
```

Comment:
We need the `key`s here, but `aliases` is `QHash<ConfigKey, Configkey>`.
And if we use `aliases` directly and do
```
if (aliases.contains(pControl->getKey())) {
    continue;
}
```
we'd skip both the original and the alias :|

Unless you have another idea I'd use my original implementation.

Reply:
Btw I'm surprised some CI runners build this.

Reply:
> we'd skip both the original and the alias :|

Oh, right. Yeah, the problem with the "lazy" solution (in terms of evaluation, not the quality of your approach ;) ) is that it depends on the iteration order whether the alias control or the actual control is hidden. I'd like to avoid that because the iteration order is dependent on the iteration order of a QHash. In this concrete case, it may be irrelevant. 

> Btw I'm surprised some CI runners build this.

I'm rather surprised some don't. Are the MacOS runners still not on Qt5.14?

Reply:
fyi the initial proposal works (at least it builds controlKeys) correctly, because the returned QHash has the original configkey as the hash key and the alias configkey as the value. Dereferencing a QHash::const_iterator instance (which is what the QSet ctor is doing) returns the value of that QHash entry. 
It's only a shame that we have to do it lazily...

Reply:
All of this would be much easier if `ControlDoublePrivate::getAllInstances();` would not return the deprecated controls...

Reply:
> Are the MacOS runners still not on Qt5.14?

IIUC we're still uing 5.12 for the Ubuntu 20.04 ppa builds :|

Reply:
> it depends on the iteration order whether the alias control or the actual control is hidden.

Yeah, I didn't care about that fact here.
If we agree to revert I'll add a comment.

Reply:
yes, pls revert to your previous solution. Mine is undoubtedly flawed.

Reply:
Alright, done.
Thanks for you review though, I learned something.